packages/twitch-pubsub-client/src/PubSubClient.ts

```diff
@@ -154,6 +155,26 @@ Register one using:
 		return this.getUserListener(user).onModAction(channel, callback);
 	}
 
+	/**
+	 * Adds a listener for arbitrary/undocumented events to the client.
+	 *
+	 * @param user The user the event will be subscribed for.
+	 * @param topic The topic to subscribe to.
+	 * @param callback A function to be called when an event is received.
+	 * @param scope An optional scope if the topic requires it.
+	 * @param channel An optional second usedid if the topic requires it.
+	 * It receives a {@PubSubMessageData} object.
```

Comment:
This comment goes on the callback param. Please also keep the empty line, as it separates the short description from the long text.